pkg/storage/segments/store.go

```diff
@@ -54,17 +54,34 @@ type Store interface {
 	List(ctx context.Context, prefix, startAfter, endBefore storj.Path, recursive bool, limit int, metaFlags uint32) (items []ListItem, more bool, err error)
 }
 
+type nodeStats struct {
+	Uptime       int
+	UptimeCount  int
+	AuditSuccess int
+	AuditCount   int
+	Excluded     storj.NodeIDList
+}
+
 type segmentStore struct {
 	oc            overlay.Client
 	ec            ecclient.Client
 	pdb           pdbclient.Client
 	rs            eestream.RedundancyStrategy
 	thresholdSize int
+	nodeStats     nodeStats
 }
 
 // NewSegmentStore creates a new instance of segmentStore
-func NewSegmentStore(oc overlay.Client, ec ecclient.Client, pdb pdbclient.Client, rs eestream.RedundancyStrategy, t int) Store {
-	return &segmentStore{oc: oc, ec: ec, pdb: pdb, rs: rs, thresholdSize: t}
+func NewSegmentStore(oc overlay.Client, ec ecclient.Client, pdb pdbclient.Client, rs eestream.RedundancyStrategy, threshold,
```

Comment:
`uptime` and `auditSuccess` are `float64`s. `uptimeCount` and `auditCount` are `int64`s